client/src/modules/Score/components/ScoreTable/index.tsx

```diff
@@ -84,9 +88,11 @@ export function ScoreTable(props: Props) {
     cityName,
     mentor,
     handleSettings: () => setIsVisibleSettings(true),
-    taskColumns: getTaskColumns(courseTasks).filter(column => !notVisibleColumns?.includes(column.name)),
+    taskColumns: getTaskColumns(courseTasks),
   });
 
+  const getVisibleColumns = (columns: any[]) => columns.filter(column => !notVisibleColumns?.includes(column.name));
```

Comment:
I wonder do we really need this as a separate function and execute it 2 times below?

we can do like this:

```typescript
const columns = getColumns({ .... })
    .filter(column => !notVisibleColumns?.includes(column.name))
```